src/components/AssignmentTexter.jsx

```diff
@@ -49,10 +50,16 @@ class AssignmentTexter extends React.Component {
 
   updateCurrentContactIndex(newIndex) {
     this.setState({
-      currentContactIndex: newIndex
+      currentContactIndex: 0
```

Comment:
why does this need to be hard-coded to `0`?  I think this is unintended/unnecessary.